apps/storefront/src/hooks/useShippingMethods.ts

```diff
@@ -1,70 +1,45 @@
-import { useState, useEffect, useCallback } from "react";
+import { useShopper } from "@ordercloud/react-sdk";
 import {
-  Address,
   IntegrationEvents,
-  OrderWorksheet,
-  RequiredDeep,
-  ShipMethod,
+  OrderShipMethodSelection,
+  ShipMethodSelection,
 } from "ordercloud-javascript-sdk";
+import { useState } from "react";
 
-interface useShippingMethodProps {
-  orderWorksheet: OrderWorksheet;
-  shippingAddress: Address;
-}
-
-export const useShippingMethods = ({
-  orderWorksheet,
-  shippingAddress,
-}: useShippingMethodProps) => {
-  const [availableMethods, setAvailableMethods] = useState<
-    RequiredDeep<ShipMethod<unknown>>[]
-  >([]);
-  const [selectedMethodIndex, setSelectedMethodIndex] = useState<string | null>(
-    null
-  );
+export const useShippingMethods = () => {
   const [error, setError] = useState<Error | null>(null);
   const [loading, setLoading] = useState(false);
 
-  const fetchShippingMethods = useCallback(async () => {
-    const orderID = orderWorksheet?.Order?.ID;
-    if (!orderID || !shippingAddress) return;
+  const { orderWorksheet } = useShopper();
 
+  const selectShipMethods = async (shipMethodID: string) => {
     setLoading(true);
+    const ShipMethodSelection: ShipMethodSelection = {
+      ShipEstimateID:
+        orderWorksheet?.ShipEstimateResponse.ShipEstimates.at(0)?.ID,
+      ShipMethodID: shipMethodID,
+    };
+    const shipMethodSelection: OrderShipMethodSelection = {
+      ShipMethodSelections: [ShipMethodSelection],
+    };
+    if (!orderWorksheet) return;
     try {
-      const worksheet = await IntegrationEvents.GetWorksheet(
+      await IntegrationEvents.SelectShipmethods(
```

Comment:
this is also on the useShopper hook - not sure if this hook needs to exist if it's refactored?